update enforce_joint_model_state_space in config

yifeifang opened the pull request:
I've encountered a problem where set_path_constraint in moveit commander with orientation constraint didn't really work (Sometimes it is still going out of the constraint). After a bit of research, I found this workaround in Moveit! See this pose for detail https://github.com/ros-planning/moveit/pull/541. With this changes in the ompl_planner.yaml it will enforce it to choose store info in joint space which will make sure the set_path_constraint interface behaves as desired. See https://www.youtube.com/watch?v=kIy_dX4ovQM&ab_channel=%E6%96%B9%E6%98%93%E9%9D%9E this video for constrain in work.

yifeifang committed:
update enforce_joint_model_state_space in config

zzhu-fetch reviewed (approved):
Looks good!

yifeifang commented:
Hi zzhu,
It looks like the PR is being reviewed and approved. Is there anything else that I need to do on my end?

erelson commented:
Hi @yifeifang, sorry, our review process here does tend to be a bit slow.  I need to review it as well but need to spend some time getting context/background (and need to find some spare time for that).  @zzhu-fetch is pretty new here, so probably shouldn't have been assigned as a reviewer ;).

One quick thought on my end: I think it would be better if the commented information in the diffs were instead in the git commit message.  Does that sound reasonable to you, @yifeifang ?

Lastly, when we do get a chance to merge this, we would probably also forward-port this change to the noetic branch `ros1`.

yifeifang commented:
Thanks for your reply! I think it's a good point and I will go ahead and move the comment in to git commit description

yifeifang committed:
update ompl_planning with enforce joint model state space set to true

This will enforce JointModelStateSpaceFactory for problem representing see
https://github.com/ros-planning/moveit/pull/541 for detail
basically this would allow path constraints in moveit_commander to work properly
